doc/api/net.md

```diff
@@ -512,14 +512,17 @@ If `true` - [`socket.connect(options[, connectListener])`][`socket.connect(optio
 haven't yet finished. Will be set to `false` before emitting `connect` event
 and/or calling [`socket.connect(options[, connectListener])`][`socket.connect(options, connectListener)`]'s callback.
 
-### socket.destroy()
+### socket.destroy([exception])
```

Comment:
@jasnell I chose `exception` because that's what it is called in the code. If we want to change it, do we have to reach into the code to change it there too? (I don't like `error` for the document because then you have to explain that `error` will be passed as an argument to any listeners on the `'error'` event, which can get confusing especially to new users. So I'd probably go with `reason` over `error`, but `reason` doesn't signal to the reader that it's an error the way `exception` does.)

As for the type, it would seem to be expecting an `Error` object but it actually doesn't look like there's any enforcement in the code so you could probably send it anything at all. I mean, it's just an argument for a listener, so it's kinda sorta up to the end user, I suppose. Suggest that it *should* (rather than *must*) be an `Error` perhaps?